app/javascript/packs/application.js

```diff
@@ -0,0 +1,22 @@
+/* eslint no-console:0 */
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+//
+// To reference this file, add <%= javascript_pack_tag 'application' %> to the appropriate
+// layout file, like app/views/layouts/application.html.erb
+
+
+// Uncomment to copy all static images under ../images to the output folder and reference
+// them with the image_pack_tag helper in views (e.g <%= image_pack_tag 'rails.png' %>)
+// or the `imagePath` JavaScript helper below.
+//
+// const images = require.context('../images', true)
+// const imagePath = (name) => images(name, true)
+
+console.log('Hello World from Webpacker')
```

Comment:
Wheee!!! Ok, so not sure if i'm jumping the gun here but I've been wondering about webpack/rails asset pipeline integration. Some possible hints on this to bear in mind:

https://duckduckgo.com/?q=webpack+asset+pipeline&atb=v121-6&ia=images

Reply:
I'm wondering if this is related to the test error we're seeing. But, the error seems relatively simple; it's asking us to fill out the `/public/packs-test/manifest.json` file with `application.js` (and possibly more than that?):

```
Minitest::UnexpectedError:         ActionView::Template::Error: Webpacker can't find application.js in /home/runner/work/plots2/plots2/public/packs-test/manifest.json. Possible causes:
        1. You want to set webpacker.yml value of compile to true for your environment
           unless you are using the `webpack -w` or the webpack-dev-server.
        2. webpack has not yet re-run to reflect updates.
        3. You have misconfigured Webpacker's config/webpacker.yml file.
        4. Your webpack configuration is not creating a manifest.
        Your manifest contains:
        {
        }
```

Is there a link to docs/guidance on the `manifest.json` file that could help us with this?

Reply:
@jywarren This link seems relevant: https://github.com/reactjs/react-rails/issues/997#issuecomment-506258423

I do have a `manifest.json` set up locally, but I haven't been committing it. I added `public/packs` (which includes `manifest.json`) to my local `.gitignore`. I'm not 100% confident on this, but I think `manifest.json` isn't something to be committed, it's supposed to be generated with the instance of the app. See this for reference: https://stackoverflow.com/questions/48416457/should-we-commit-manifest-json-file-to-git-repo-when-using-rails-webpacker

I think what's supposed to happen is `rails generate` React commands have to be run on each particular instance of the app?

Reply:
I see, ok, thanks!

I'll try this out but i have to run soon, so one thing we might do is use this `rails g webpack:install; rails g webpack:install:react; rails g react:install` on all the lines I added to `tests.yml` and `gitpod.yml`.

Reply:
OK i tried this but didn't remove the prior `      run: ./public/lib/.bin/webpack-dev-server` -- but i can remove that too if it causes problems.